File: C#Integration/PokerProtocol/Commands/Game/TableClosedCommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using EricUtility;

namespace PokerProtocol.Commands.Game
{
    public class TableClosedCommand : AbstractCommand
    {
        protected override string CommandName
        {
            get { return TableClosedCommand.COMMAND_NAME; }
        }
        public static string COMMAND_NAME = "gameTABLE_CLOSED";


        public TableClosedCommand(StringTokenizer argsToken)
        {
        }

        public TableClosedCommand()
        {
        }
    }
}

File: JAVA/PokerProtocol/src/bluffinmuffin/protocol/commands/game/TableClosedCommand.java
package bluffinmuffin.protocol.commands.game;

import java.util.StringTokenizer;

import bluffinmuffin.protocol.commands.AbstractCommand;

public class TableClosedCommand extends AbstractCommand
{
    @Override
    protected String getCommandName()
    {
        return TableClosedCommand.COMMAND_NAME;
    }
    
    public static String COMMAND_NAME = "gameTABLE_CLOSED";
    
    public TableClosedCommand(StringTokenizer argsToken)
    {
    }
    
    public TableClosedCommand()
    {
    }
}
